<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_add_values</class>
 <widget class="QWidget" name="w_add_values">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>627</width>
        <height>394</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Добавление новой записи в базу данных</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Наименование организации</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_name_org">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Модель катриджа</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_model_catridge">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID катриджа</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_id_catridge">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Выполненные работы</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_vip_rabot">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>431</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_zhalobi">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>431</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Жалобы со слов клиента</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_commentarii">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>230</y>
         <width>431</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Комментарии</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>330</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата ремонта</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="data_remonta">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>360</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_add_new_data">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить запись</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_sn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>627</width>
        <height>394</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Добавление записи с обновлением данных</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Наименование организации</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_name_org">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_add_update_data">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить запись</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID катриджа</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_le_id_catridge">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>21</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_catridge">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="update_txt_model_catridge">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Модель катриджа</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_sn_catridge">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SN катриджа</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="update_data_remonta">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>360</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="update_txt_commentarii">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>230</y>
         <width>421</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="update_txt_vip_rabot">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>421</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Жалобы со слов клиента</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Комментарии</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="update_txt_zhalobi">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>421</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>330</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата ремонта</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Выполненные работы</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
